federation/client/clientset_generated/federation_internalclientset/typed/core/unversioned/fake/fake_core_client.go

```diff
@@ -46,8 +47,8 @@ func (c *FakeCore) Services(namespace string) unversioned.ServiceInterface {
 	return &FakeServices{c, namespace}
 }
 
-// GetRESTClient returns a RESTClient that is used to communicate
+// RESTClient returns a RESTClient that is used to communicate
 // with API server by this client implementation.
-func (c *FakeCore) GetRESTClient() *restclient.RESTClient {
-	return nil
+func (c *FakeCore) RESTClient() restclient.Interface {
+	return &fake.RESTClient{}
```

Comment:
I'm not convinced it's correct to return `fake.RESTClient{}` here. If a test calls other methods of FakeCore, this fake restClient is not triggered, so it doesn't seem to be useful to me. Can we just return nil here?